DigitalLearningSolutions.Data.Tests/Services/RegistrationServiceTests.cs

```diff
@@ -955,36 +891,11 @@ public void PromoteDelegateToAdmin_updates_existing_admin_if_inactive_admin_at_s
                         categoryId
                     )
                 ).MustHaveHappenedOnceExactly();
-            }
-        }
-
-        [Test]
-        public void PromoteDelegateToAdmin_calls_data_service_with_expected_value()
-        {
-            // Given
-            const int userId = 123;
-            var delegateUser = UserTestHelper.GetDefaultDelegateUser();
-            var adminRoles = new AdminRoles(true, true, true, true, true, true, true);
-
-            A.CallTo(() => userDataService.GetDelegateUserById(A<int>._)).Returns(delegateUser);
-            A.CallTo(() => userDataService.GetAdminUserByEmailAddress(A<string>._)).Returns(null);
-            A.CallTo(() => userDataService.GetUserIdFromDelegateId(delegateUser.Id)).Returns(userId);
-
-            // When
-            registrationService.PromoteDelegateToAdmin(adminRoles, 1, delegateUser.Id);
-
-            // Then
-            using (new AssertionScope())
-            {
                 A.CallTo(
                     () => registrationDataService.RegisterAdmin(
                         A<AdminRegistrationModel>.That.Matches(

```

Comment:
Probably easiest to just ignore the exact params here as the method should not get called at all, regardless of the params.